provisioning/library/get_perl_modules.yml

```diff
@@ -51,8 +51,9 @@
       #   simple string, and "<pkg> in <string>" matches substrings of
```

Comment:
Add a comment in the comment block (in proper order) describing the new regex_replace line for stripping out leading whitespace